.travis.yml

```diff
@@ -23,6 +23,7 @@ before_install:
 install:
   - python setup.py install
   - pip install -r requirements.txt
+  - pip install mysql-connector
```

Comment:
This should go in the [setup.py install requires](https://github.com/ISISScientificComputing/autoreduce/blob/master/setup.py#L21)